packages/@react-stately/data/docs/useAsyncList.mdx

```diff
@@ -174,6 +174,54 @@ let list = useAsyncList({
 });
 ```
 
+## Filtering
+
+There are many instances where your list of data should be filtered, such as user lookup or query searches.
+For server side filtering, this can be implemented by levering the `filterText` passed to the `load` as an API parameter.
+`useAsyncList` then returns the state variable `filterText` and its setter `setFilterText`, allowing you to update the
+`filterText` used within `load` in response to any updates within your component.
+
```

Comment:
Wonder if I should include a client side filtering section...

Probably would be along the lines of "Client side filtering can be done by filtering the items returned by `useAsyncList` yourself." with an example